packages/web-app-files/src/components/AppBar/SelectedResources/BatchActions.vue

```diff
@@ -84,5 +87,14 @@ export default {
     display: flex;
     gap: var(--oc-space-small);
   }
+
+  &-squashed {
```

Comment:
Why only for squashed state? IMO we could do the same for when the sidebar is closed.